src/urllib3/connectionpool.py

```diff
@@ -64,29 +72,31 @@ class ConnectionPool:
        target URIs.
     """
 
-    scheme = None
+    scheme: Optional[str] = None
     QueueCls = queue.LifoQueue
 
-    def __init__(self, host, port=None):
-        if not host:
+    def __init__(self, host: Optional[str], port: Optional[int] = None) -> None:
```

Comment:
Any reason to have `Optional[str]` instead of `str` if we always error on this case?

Reply:
changed to str